ansible/group_vars/all

```diff
@@ -209,15 +209,14 @@ nginx:
   version: "{{ nginx_version | default('1.13') }}"
   port:
     http: 80
-    api: 443
+    https: 443
   ssl:
-    path: "{{ openwhisk_home }}/ansible/roles/nginx/files"
-    cert: "openwhisk-server-cert.pem"
-    key: "openwhisk-server-key.pem"
-    password_enabled: false
```

Comment:
it's not necessary to have both password_enabled and the password_file parameter - this chance consolidates both.